milc/milc.py

```diff
@@ -56,7 +56,7 @@ def __init__(self):
 
         # Figure out our program name
         self.prog_name = sys.argv[0][:-3] if sys.argv[0].endswith('.py') else sys.argv[0]
-        self.prog_name = self.prog_name.split('/')[-1]
+        self.prog_name = os.environ.get('MILC_APP_NAME', os.path.basename(self.prog_name))
```

Comment:
This is just minor improvement, although I believe without it, this can break on Windows.